### PR TITLE
Provider model

### PR DESCRIPTION
**Pull request by dczulada**
The Provider model is "now" in cqm-models. Use this structure to export.
https://github.com/projecttacoma/cqm-models/pull/93

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] Code diff has been done and been reviewed
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass

**Reviewer 1:**

Name: @mayerm94 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

**Reviewer 2:**

Name: @okeefm 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

### CODE REVIEW

File: lib/html-export/qdm-patient/qdm_patient.mustache [hunk 1]
```diff
@@ -2,12 +2,15 @@
 <html xmlns:n1="urn:hl7-org:v3" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
   <head>
     <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+    {{#patient}}
     <title>Cypress Certification Patient Test Record: {{{given_name}}} {{{family_name}}}</title>
```
Comment (mayerm94) [inline lib/html-export/qdm-patient/qdm_patient.mustache]: Should this be changed to familyName?

Comment (dczulada) [reply lib/html-export/qdm-patient/qdm_patient.mustache]: yes
